subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_lte_notification.c

```diff
@@ -17,6 +17,10 @@
 
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(lwm2m_neighbour_cell, CONFIG_LWM2M_CLIENT_UTILS_LOG_LEVEL);
+static void lte_notify_handler(const struct lte_lc_evt *const evt);
+
+static bool event_handler_requested;
```

Comment:
These 2 should be in the `#ifdef(CONFIG_LWM2M_CLIENT_UTILS_NWK_REG_NOTIFICATION)`

Reply:
No these 2 are needed for both ncell and Network registration notify.

Reply:
Oh yeah, true.